api-doc.md

```diff
@@ -298,3 +298,42 @@
             </url>
         </urlset>
 ```
+
+### 4. Send New Leave Request mail
+
+```
+- Method : POST
+- Endpoint : /api/leave/new
+- Description : API for sending mail to HR for New leave request
+- Request:
+    - Headers : none
+    - Body :
+        {
+            "name": "mansi dhameliya",
+            "date": "3 jan 2023",
```

Comment:
How to send date for multiple leaves, like leave for 3 days?

Reply:
Sir told me that only one date will be used for the date of **application(applied date)** of display leave request, and other information will be displayed in unity.